### ISSUE TITLE
is it possible to display thumbnails HORIZONTALLY?

### ISSUE THREAD

**Issue by cordoba3030** [react-pdf-viewer/react-pdf-viewer#1047]
Hi 
Is it possible to display thumbnails HORIZONTALLY? It will be cool on magazine view (like as Joomag, Flippingbook, Issue, etc).

Dou plan to add feature that shos thumbnails horizontally?

**Comment by phuoc-ng** [comment]
@cordoba3030 
Do you mean something likes this https://github.com/react-pdf-viewer/react-pdf-viewer/issues/291#issuecomment-706706691?

**Comment by cordoba3030** [comment]
Hi
I mean magazine view

Pages view is good, I mean thumbnails place (design) - on bottom -
horizontally... Now they are on the left sidebar - vertically, I mean like
an attached reader if thumbnails are situated on the bottom - horizontally
it will be a good alternative second viewer version...  Like Flowpaper,

OR

like Flowpaper, Joomag, Flippingbook viewer
https://viewer.joomag.com/mobile-living-made-easy-australia-issue-1/0316456001519003955

https://online.flippingbook.com/view/485518472/



вс, 13 мар. 2022 г. в 08:13, phuoc-ng ***@***.***>:

**Comment by phuocng** [comment]
@cordoba3030 This request was implemented in the latest code and will be included in the next version (v3.11.0).

## API

```tsx
import { thumbnailPlugin, ThumbnailDirection } from '@react-pdf-viewer/thumbnail';

const thumbnailPluginInstance = thumbnailPlugin();
const { Thumbnails } = thumbnailPluginInstance;

// Display thumbnails horizontally
<Thumbnails thumbnailDirection={ThumbnailDirection.Horizontal} />

// Display thumbnails vertically
<Thumbnails thumbnailDirection={ThumbnailDirection.Vertical} />
```

## Demo

The following demo uses this [sample code](https://github.com/react-pdf-viewer/react-pdf-viewer/blob/master/demo/nextjs/pages/magazine/index.tsx).

_Magazine layout with normal viewmode_

https://user-images.githubusercontent.com/57786711/214833188-e5c53adb-01a8-41fc-95cb-90bf07a96e30.mp4

_Magazine layout with dual page viewmode_

https://user-images.githubusercontent.com/57786711/214833112-366e0544-f6da-4abe-affb-932710ea4d4c.mp4

**Comment by phuocng** [comment]
Please upgrade to [v3.11.0](https://react-pdf-viewer.dev/updates/). I create a live example as well
https://react-pdf-viewer.dev/examples/magazine-layout/
